packages/orchestrator-ui-components/src/components/WfoWorkflowSteps/WfoStep/WfoStepForm.tsx

```diff
@@ -27,16 +29,28 @@ export const WfoStepForm = ({
             return Promise.reject();
         }
 
-        return resumeProcess({ processId, userInputs: processInput }).then(
-            () => {
+        return resumeProcess({ processId, userInputs: processInput })
+            .unwrap()
+            .then(() => {
                 setIsProcessing(true);
-            },
-        );
+            })
+            .catch((error) => {
+                if (error?.status !== HttpStatus.FormNotComplete) {
+                    if (error?.status === HttpStatus.BadRequest) {
```

Comment:
If you put this predicated as negated in the if above (line 38) you would end up with the same functionality, right?